Statistics.Test/StatsUnitTest.cs

```diff
@@ -1,30 +1,39 @@
 using System;
 using Xunit;
 using Statistics;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
 
 namespace Statistics.Test
 {
     public class StatsUnitTest
     {
+        
         [Fact]
         public void ReportsAverageMinMax()
         {
             var statsComputer = new StatsComputer();
             var computedStats = statsComputer.CalculateStatistics(
-                new List<___>{1.5, 8.9, 3.2, 4.5});
+                new List<double>{1.5, 8.9, 3.2, 4.5});
             float epsilon = 0.001F;
-            Assert.True(Math.Abs(statsComputer.average - 4.525) <= epsilon);
-            Assert.True(Math.Abs(statsComputer.max - 8.9) <= epsilon);
-            Assert.True(Math.Abs(statsComputer.min - 1.5) <= epsilon);
+            Assert.True(Math.Abs(computedStats.Average - 4.525) <= epsilon);
+            Assert.True(Math.Abs(computedStats.Max - 8.9) <= epsilon);
+            Assert.True(Math.Abs(computedStats.Min - 1.5) <= epsilon);
         }
+        [Fact]
         public void ReportsNaNForEmptyInput()
         {
             var statsComputer = new StatsComputer();
             var computedStats = statsComputer.CalculateStatistics(
-                new List<___>{});
+                new List<double>{});
+            Assert.True(double.IsNaN(computedStats.Average));
+            Assert.True(double.IsNaN(computedStats.Max));
+            Assert.True(double.IsNaN(computedStats.Min));
```

Comment:
This is a good test. Can you extend the tests here? What if the input itself contains NaN? Write the test and then do the implementation.